config/default.yml

```
@@ -711,7 +711,9 @@ Rails/Pluck:
   Description: 'Prefer `pluck` over `map { ... }`.'
   StyleGuide: 'https://rails.rubystyle.guide#pluck'
   Enabled: 'pending'
+  SafeAutoCorrect: false
```

This looks like false positives indistinguishable from an alias.

```suggestion
  Safe: false
```